modules/rules_testing/metadata.json

```diff
@@ -4,14 +4,15 @@
         {
             "email": "rlevasseur@google.com",
             "github": "rickeylev",
-            "name": "Richard Levasseur"
+            "name": "Richard Levaseur"
```

Comment:
@rickylev Is this a typo?

Reply:
Yes, I've fixed it upstream. You can close this or merge it as-is; a 0.0.3 build will have it fixed.

Reply:
Ah, looks like I have some other misconfigurations to fix. I'll delete this branch; you can close the PR